examples/dreambooth/train_dreambooth.py

```diff
@@ -447,13 +470,23 @@ def get_full_repo_name(model_id: str, organization: Optional[str] = None, token:
 def main(args):
     logging_dir = Path(args.output_dir, args.logging_dir)
 
+    gpu = int(args.device.split(":")[1]) if args.device != "cuda" else "cuda"
+    if torch.cuda.is_available():
+        torch.cuda.set_device(gpu)
```

Comment:
don't think we should have this here, this will break multi-gpu. We can set the device using `CUDA_VISIBLE_DEVICES` env variable.  Which is also recommended by torch, cf https://pytorch.org/docs/stable/generated/torch.cuda.set_device.html